### PR TITLE
feat: [IA-382] Added a label read by screen reader

### PR DESCRIPTION
**Pull request by filippo-tenaglia**

### CODE REVIEW

File: locales/it/index.yml [hunk 1]
```diff
@@ -65,6 +65,7 @@ global:
     moreInfo: più info
   accessibility:
     reload: "Ricarica"
+    switchLabel: "Selezionare per attivare"
```
Comment (thisisjp) [inline locales/it/index.yml:68]:
```suggestion
    switchLabel: "Seleziona per attivare"
```

**Review by thisisjp — COMMENTED** [review]
@filippo-tenaglia we usually write these hints using the imperative verb, rather than the infinitive—check my suggested change. Sounds ok for you? 😊

File: ts/components/ui/Switch.tsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import { NativeBase, Switch as NBSwitch } from "native-base";
+    import { NativeBase, Switch as NBSwitch } from "native-base";
```
Comment (pietro909) [inline ts/components/ui/Switch.tsx:1]: Mind the leading whitespaces :-)

File: ts/components/ui/Switch.tsx [hunk 3]
```diff
@@ -12,14 +13,18 @@ const maybeDisabled = fromPredicate(
  * NativeBase Switch component styled with the app's brand primary color
  */
 export default class Switch extends React.Component<NativeBase.Switch> {
-  public render() {
+  public render(accessibleLabelText) {
```
Comment (pietro909) [inline ts/components/ui/Switch.tsx:16]: The `render` function doesn't accept any argument: it should use `this.props` and `this.state` as the only input sources. This is also the reason we the build is failing.

Please see also [the official documentation](https://reactjs.org/docs/react-component.html#render).

File: ts/components/ui/Switch.tsx [hunk 4]
```diff
@@ -12,14 +13,18 @@ const maybeDisabled = fromPredicate(
...
     const thumbColor: string = maybeDisabled(this.props.disabled)
       .map(_ => variables.brandPrimaryLight)
       .getOrElse(variables.contentPrimaryBackground);
 
     return (
       <NBSwitch
-        // Stick
+accessible={true}
+// added accessibleLabel to manage the text read by screen readers
+// this label can be a default value or received from external call
```
Comment (pietro909) [inline ts/components/ui/Switch.tsx:25]: We don't usually write comments about well-known or already documented behavior (ie: how optional properties work).

**Review by pietro909 — CHANGES_REQUESTED** [review]
I suggest you run `yarn tsc:noemit` on your machine to ensure that the code is building, and perform an actual test locally to verify that the PR is addressing the issue properly.

**Review by pietro909 — APPROVED** [review]

**Pull request merged**